platform/core.windows/manifest.mf

```diff
@@ -6,5 +6,5 @@ OpenIDE-Module-Provides: org.netbeans.core.WindowSystem, org.openide.windows.Win
 OpenIDE-Module-Recommends: org.netbeans.core.windows.nativeaccess.NativeWindowSystem
 AutoUpdate-Show-In-Client: false
 AutoUpdate-Essential-Module: true
-OpenIDE-Module-Specification-Version: 2.92
+OpenIDE-Module-Specification-Version: 2.91
```

Comment:
I guess I would suggest to actually increase the spec version (so that the spec versions are monotonically increasing). I.e.:
```suggestion
OpenIDE-Module-Specification-Version: 2.93
```